index.html

```diff
@@ -5,194 +5,263 @@
+    <link rel="stylesheet" media="screen and (min-width: 601px)" href="desktop.css">
+    <title>Ayomide S.E.P</title>
 </head>
 <body>
-    <section>
-        <header class="header">
-            <nav class="navbar">
-                <div class="logo">
-                    <img src="./assets/My Logo.png" alt="">
-                </div>
-                <div class="hamburger">
-                    <span class="line-top"></span>
-                    <span class="line-middle"></span>
-                    <span class="line-bottom"></span>
-                </div>
-            </nav>
-            <div class="first intro">
-                <h1 class="title">
-                    I'm Ayomide.s.e.p<br>
-                    Glad to see you!
+    <header class="header">
+        <nav class="nav pad-aside">
+            <a href="#">
+                <img src="./assets/Logo.png" alt="fir">
+            </a>
+            <a href="#"><img src="./assets/hamburger.png" class="hamburger" alt="se"></a>
+            <div class="nav-link-desktop">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+        <section class="intro">
+            <div class="introc-content pad-aside">
+                <h1 class="pad-aside">
+                    I am Ayomide.
+                Glad to see you here!
                 </h1>
-                <p class="bio">
-                    I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                <p class="pad-aside">
+                    I am a software developer! I can help you build a product , feature or website   Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded,
+                    do not hesitate to contact me.
                 </p>
-                <h3 class="let-connect">
+                <h3 class="pad-aside">
                     LET'S CONNECT
                 </h3>
-                <ul class="social-icons">
-                    <li><a href="#"><i class="fa-brands fa-twitter fa-lg"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin fa-lg"></i></a></li>
-                    <li><a href="#"><i class="fa-solid fa-m fa-lg"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-github fa-lg"></i></a></li>
-                    <li><a href="#"><i class="fa-regular fa-hand-peace fa-flip-horizontal fa-lg"></i></a></li>
-                </ul>
+                <div class="social-icons pad-aside ">
+                    <img src="./assets/social icons/Vector.svg" alt="">
+                    <img src="./assets/social icons/Linkedin icon.svg" alt="">
```

Comment:
- [ ] Kindly reduce the top margin above the header, to match the Figma design.

- Your design
<img width="960" alt="image" src="https://user-images.githubusercontent.com/41163571/236342383-bfba2b99-9c90-4941-b432-fac9411d5cb7.png">

- Figma design
<img width="960" alt="image" src="https://user-images.githubusercontent.com/41163571/236341746-43f8d6b3-93c4-43ff-b5cd-1be40a4585dd.png">

Reply:
Thank you. I miscalculated, instead of 284px I used 359px.
Thank you😎